Update memory-integrity.md (#5439)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/device-guard/memory-integrity.md

- [Microsoft Defender Advanced Threat Protection (Microsoft Defender ATP)](https://go.microsoft.com/fwlink/p/?linkid=2069559)

> Memory integrity is a feature of Windows that ensures that code that can run in Windows' kernel is securely designed and trustworthy.  It is  powerful security boundary that leverages hardware virtualization and Hyper-V to protect Windows kernel mode processes against injection and execution of malicious or unverified code.  The integrity of code that runs on Windows is validated by Memory integrity so that Windows is resistant to attack from malicious software. Memory integrity helps block many types of malware from running in Windows 10 and Windows Server 2016 environments.

Reviewer: Suggesting some language flow improvements:
```suggestion
Memory integrity is a feature of Windows, ensuring that code running in the Windows kernel is securely designed and trustworthy.  It is a powerful security boundary that leverages hardware virtualization and Hyper-V to protect Windows kernel mode processes against injection and execution of malicious or unverified code. The integrity of the code that runs on Windows is validated by Memory integrity, making Windows resistant to attack from malicious software. Memory integrity helps blocking many types of malware from running in Windows 10 and Windows Server 2016 environments.
```
(feel free to adjust my suggestions if you find that some of them are incorrect or imprecise)

BTW, @JohanFreelancer9 may have other suggestions than mine to improve the syntax & grammar.

Reviewer: @marcusrussell : Please ignore this suggestion (mark this conversation as resolved by clicking the button "Mark conversation resolved" below).

I defer to the suggested change below for the actual requested changes.

> Memory integrity is a feature of Windows that ensures that code that can run in Windows' kernel is securely designed and trustworthy.  It is  powerful security boundary that leverages hardware virtualization and Hyper-V to protect Windows kernel mode processes against injection and execution of malicious or unverified code.  The integrity of code that runs on Windows is validated by Memory integrity so that Windows is resistant to attack from malicious software. Memory integrity helps block many types of malware from running in Windows 10 and Windows Server 2016 environments.

Reviewer 2: ```suggestion
Memory integrity is a feature of Windows which ensures that code that can run in Windows' kernel is securely designed and trustworthy.  It is a powerful security boundary that leverages hardware virtualization and Hyper-V to protect Windows kernel mode processes against injection and execution of malicious or unverified code.  The integrity of code that runs on Windows is validated by Memory integrity so that Windows is resistant to attack from malicious software. Memory integrity helps block many types of malware from running in Windows 10 and Windows Server 2016 environments.
```
